data_prep/roberta_graph_dataset.py

```diff
@@ -45,7 +45,7 @@ def _generate_features(self):
                 docs = self._raw_texts[i * batch_size:max_docs]
                 encodings = self._tokenizer(docs, truncation=True, padding=True)
                 items = {k: torch.tensor(v, dtype=torch.long, device=self._device) for k, v in encodings.items()}
-                out, _ = doc_embedder(items)
+                _, out = doc_embedder(items['input_ids'])
```

Comment:
Isn't this using the labels our `RobertaEncoder` returns? (see image)

![Screenshot 2021-05-21 at 16 03 40](https://user-images.githubusercontent.com/8261902/119149789-1d7ed980-ba4e-11eb-8eea-e33b5d548bd4.png)

Reply:
Ok, I see you are now not using our `RobertaEncoder` anymore. Then this is obsolete

Reply:
No we still do use RobertaEncoder. But only if the path to that model is specified. I changed it now with an if statement